src/Css/Stylesheet.php

```diff
@@ -1612,39 +1612,26 @@ private function _parse_font_face($str)
      */
     private function _parse_properties($str)
     {
-        $properties = preg_split("/;(?=(?:[^\(]*\([^\)]*\))*(?![^\)]*\)))/", $str);
         $DEBUGCSS = $this->_dompdf->getOptions()->getDebugCss();
 
-        if ($DEBUGCSS) {
-            print '[_parse_properties';
-        }
+        if ($DEBUGCSS) print '[_parse_properties';
```

Comment:
Should we go ahead and add brackets even if the condition is a single line (just when changing relevant code)?

Reply:
Done. Just wanted to make the statement consistent with the other debug lines within the method, but I agree.